### DISCUSSION TITLE
What happened to foalcon booru?

### DISCUSSION THREAD

**Discussion by Josh167** [rr-/szurubooru#575] [General]
It went down for 4 days, and when it came back online, it's like it forgot EVERYTHING! It thinks I'm an administator(which I'm not) and it's acting like the one pic I posted on it is the very first one it's ever got. What happened to it?

**Comment by TiredSysOp** [comment]
Sounds like it got restarted or upgraded and it's not connecting to it's old database, if you've created an account on it and it's the only one then it'll automatically make you an admin. Contact the owner

**Reply by Josh167** [reply]
How do I contact the owner? I don't even know who owns it.

**Reply by TiredSysOp** [reply]
If I found the right one it looks like the backend is down. Have another look and see if it says anything about not being able to fetch basic configuration

**Reply by Josh167** [reply]
Does that mean the owner's trying to fix it? Also, yes; I did find the message saying it can't fetch basic configuration from the server. The website is https://booru.foalcon.com.

**Reply by TiredSysOp** [reply]
I hope so, https://foalcon.com/index.html if that's the right community drop into their discord and have a quiet word.

**Reply by Josh167** [reply]
I'm trying to mention it, but apparently the only channels I can access are "rules", "Introductions", and "Public Ban History". There's no help channel on there, and I'm not getting any response from either of them.

**Comment by Josh167** [comment]
You know what? I don't understand the whole point anymore. No one's answering my comments on that discord server, and I'm on the verge of going insane..

**Reply by TiredSysOp** [reply]
I wouldn't worry about it mate, too much stress and worry. If it comes back then it comes back. Otherwise fire up your own one if it's not back in a couple days. 💪

**Comment by noirscape** [comment]
I... don't know if this is the right place to discuss the uptime of an instance?

**Reply by TiredSysOp** [reply]
It isn't but the problem has solved itself now anyway. But it's a sneaky way of finding out how Szurubooru operates under less than ideal circumstances. If you've got the perms for it you can nuke this thread 💪
